apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/paidWorks.js

```diff
@@ -0,0 +1,159 @@
+const dayjs = require('dayjs')
+const { get } = require('lodash')
+const { rubles } = require('rubles')
+
+const { FinanceInfoClient } = require('@open-condo/clients/finance-info-client/FinanceInfoClient')
+const { getById, getByCondition } = require('@open-condo/keystone/schema')
+
+const { buildExportFile, DOCX_FILE_META } = require('@condo/domains/common/utils/createExportFile')
+const { buildUploadInputFrom } = require('@condo/domains/common/utils/serverSchema/export')
+const { normalizeTimeZone } = require('@condo/domains/common/utils/timezone')
+const { DEFAULT_INVOICE_CURRENCY_CODE, INVOICE_STATUS_CANCELED } = require('@condo/domains/marketplace/constants')
+const { Invoice } = require('@condo/domains/marketplace/utils/serverSchema')
+const { DEFAULT_ORGANIZATION_TIMEZONE } = require('@condo/domains/organization/constants/common')
+const { TICKET_DOCUMENT_GENERATION_TASK_FORMAT } = require('@condo/domains/ticket/constants/ticketDocument')
+
+
+const DATE_FORMAT = 'DD.MM.YYYY'
+const formatDate = (date, timeZone, format = DATE_FORMAT) => {
+    return dayjs(date).tz(timeZone).format(format)
+}
+
+const buildExportWordFile = async ({ task, documentData, locale, timeZone }) => {
+    const { id, ticket } = task
+
+    const { stream } = await buildExportFile({
+        templatePath: `./domains/ticket/templates/ticketDocumentGenerationTemplates/paidWorks/${locale}.docx`,
+        replaces: documentData,
+    })
+
+    return {
+        stream,
+        filename: `paid_works_ticket_${ticket.id}_${formatDate(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
+        mimetype: DOCX_FILE_META.mimetype,
+        encoding: DOCX_FILE_META.encoding,
+        meta: {
+            listkey: 'TicketDocumentGenerationTask',
+            id,
+        },
+    }
+}
+
+const renderMoney = (amount, currencyCode, locale) => {
+    const options = { currency: currencyCode }
+    const numberFormat = new Intl.NumberFormat(locale, options)
+    const parts = numberFormat.formatToParts(amount)
+    return parts.map((part) => part.value).join('')
+}
+
+
+const generateTicketDocumentOfPaidWorks = async ({ task, baseAttrs, context, locale, ticket, organization }) => {
+    const dadata = new FinanceInfoClient()
+    let psrn = null, organizationAddress = null, iec = null
+    try {
+        ({ iec, psrn, organizationAddress } = await dadata.getOrganization(organization.tin))
+    } catch (error) {
+        console.error(error)
+    }
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone).locale(locale)
+
+    const contact = ticket.contact
+        ? await getById('Contact', ticket.contact)
+        : null
+    const employee = organization.id && ticket.executor
```

Comment:
what if employee is deleted?

Reply:
also, why do you need to check organization.id existence ?

Reply:
Since the same employee can be connected to several organizations. Therefore, the condition must be both the user ID and organization ID in which he is an employee